kafka-handler/src/test/org/apache/hadoop/hive/kafka/KafkaStorageHandlerInfoTest.java

```diff
@@ -0,0 +1,133 @@
+
+    private static final String RESULT = "Partition(topic = TEST1, partition = 0, leader = 0, replicas = [0], isr = [0], offlineReplicas = []) [start offset = [0], end offset = [" + RECORD_NUMBER + "]]";
+
+    @BeforeClass
+    public static void setupCluster() throws Throwable {
+        KAFKA_BROKER_RESOURCE.before();
+        sendData();
+    }
+
+    @AfterClass
+    public static void tearDownCluster() {
+        KAFKA_BROKER_RESOURCE.deleteTopic(TOPIC);
+        KAFKA_BROKER_RESOURCE.after();
+    }
+
+    @Before
+    public void setUp() {
+        LOG.info("setting up Config");
+        setupConsumer();
+    }
+
+    @After
+    public void tearDown() {
+        consumer.close();
+        LOG.info("tearDown");
+    }
+
+    private void setupConsumer() {
+        this.conf.set(KafkaTableProperties.HIVE_KAFKA_BOOTSTRAP_SERVERS.getName(), KafkaBrokerResource.BROKER_IP_PORT);
+        consumerProps = new Properties();
+        consumerProps.setProperty(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "false");
+        consumerProps.setProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "none");
+        consumerProps.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, KafkaBrokerResource.BROKER_IP_PORT);
+        consumerProps.setProperty(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
+        consumerProps.setProperty(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
+        //The configuration values are not default and are given randomly , these can be changed and tested if required
+        consumerProps.setProperty(ConsumerConfig.REQUEST_TIMEOUT_MS_CONFIG, "3002");
+        consumerProps.setProperty(ConsumerConfig.FETCH_MAX_WAIT_MS_CONFIG, "3001");
+        consumerProps.setProperty(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "3001");
+        consumerProps.setProperty(ConsumerConfig.METADATA_MAX_AGE_CONFIG, "100");
+        this.consumer = new KafkaConsumer<>(consumerProps);
+    }
+
+    @Test
+    public void testFormatAsText() {
+        KafkaStorageHandlerInfo storageHandlerInfo = new KafkaStorageHandlerInfo(TOPIC, consumerProps);
+        Assert.assertEquals(String.class, storageHandlerInfo.formatAsText().getClass());
+        String text = storageHandlerInfo.formatAsText();
+        Assert.assertEquals(RESULT, text);
+    }
+
+    private static void sendData() {
+        LOG.info("Setting up kafka producer");
+        Properties producerProps = new Properties();
+        producerProps.setProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, KafkaBrokerResource.BROKER_IP_PORT);
+        producerProps.setProperty(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.ByteArraySerializer");
+        producerProps.setProperty(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.ByteArraySerializer");
+        producer = new KafkaProducer<>(producerProps);
+        LOG.info("kafka producer started");
```

Comment:
nit: if you want to keep on using capital. first letters, this might want to become "Kafka producer started"